MerlinAU.sh

```diff
@@ -4288,6 +4288,7 @@ _AddCronJobEntry_()
        "$newSetting" && \
        Update_Custom_Settings FW_New_Update_Cron_Job_Schedule "$newSchedule"
    fi
+   _ReleaseLock_
```

Comment:
To what I can tell, this specific release lock isn't required. Any recommendations? Keep it for safety or remove it as junk?

Reply:
Done!